index.js

```diff
@@ -57,6 +63,37 @@ app.get('/products/:Id/description-and-standards', (req, res) => {
   });
 });
 
+app.delete('/products/:Id', (req, res) => {
+  const productId = req.params.Id;
+  con.query(`DELETE FROM StandardsandDescriptions WHERE Product_id = ${productId}`, (SandDDeleteErr) => {
+    if (SandDDeleteErr) throw SandDDeleteErr;
+    con.query(`DELETE FROM Product WHERE ID = ${productId}`, (prodDeleteErr, prodDeleteRes) => {
+      if (prodDeleteErr) throw prodDeleteErr;
+      res.send(prodDeleteRes);
+    });
+  });
+});
+
+app.post('/products/add', (req, res) => {
+  const {
+    productDescription, pageLength, answerKeyIncluded, teachingDuration,
+  } = req.body;
+  const query = `INSERT INTO Product (productDescription, pageLength, answerKeyIncluded, teachingDuration) VALUES (${productDescription}, ${pageLength}, ${answerKeyIncluded}, ${teachingDuration})`;
+  con.query(query, (prodPostErr) => {
+    if (prodPostErr) throw prodPostErr;
+    res.json(req.body);
+  });
+});
+
+// TODO: need a way to look up product by name or other identifier than description
+app.put('/products/updateProductDescription', (req, res) => {
+  const { productDescription, id } = req.body;
```

Comment:
I see you have the id identifies in the req.body, but i think i'd put the id in the endpoint as well

Reply:
I guess I was thinking it would be a form data, so the user doesn't navigate to a product-specific URL. Does that make sense?